src/app/components/Search/index.tsx

```diff
@@ -99,11 +102,22 @@ const SearchCmp: FC<SearchProps> = ({ scope, variant, disabled, onFocusChange: o
   const searchPlaceholderTranslated = isMobile ? t('search.mobilePlaceholder') : t('search.placeholder')
   const [value, setValue] = useState('')
   const [isFocused, setIsFocused] = useState(false)
+  const [isTooShort, setIsTooShort] = useState(false)
```

Comment:
Can you separate "isTooShort" into a small PR

Reply:
Yes but first I want to change the UX according to @donouwens's suggestion about the search suggestions.

Reply:
See #671